social_django/compat.py

```diff
@@ -3,9 +3,9 @@
 import django
 from django.db import models
 
-if django.VERSION >= (2, 0):
+try:
     from django.urls import reverse
-else:
+except ImportError:
     from django.core.urlresolvers import reverse
 
 if django.VERSION >= (1, 10):
```

Comment:
I know this is old code, but how about making it Pythonic too?

Reply:
It's not clear for me the meaning of your question, I think using exception is better than read the version.

Anyway, I tried to avoid annoying deprecation warning messages when I use `social-app-django` with Django 1.10 or 1.11 and `python -Wd manage.py runserver`

Reply:
I was referring to the other version checks in this file, e.g.:
```
if django.VERSION >= (1, 10):
```
So that it's consistent.  :)